fx-src/FirefoxStorage.js

```diff
@@ -299,6 +302,40 @@ export default class FirefoxAdapter extends BaseAdapter {
         return result[0].getResultByName("last_modified");
       });
   }
+
+  /**
+   * Reset the sync status of every record and collection we have
+   * access to.
+   */
+  resetSyncStatus() {
```

Comment:
Yes, you're right!
So far we can't enumerate the local databases with IndexedDB : https://bugzilla.mozilla.org/show_bug.cgi?id=934640